README.md

```diff
@@ -1,55 +1,104 @@
-# COVID-19 CLI
+# CoVID-19 CLI
 
 <img src="https://badges.frapsoft.com/os/v1/open-source.png?v=103"> <a href="LICENSE"><img alt="GitHub" src="https://img.shields.io/github/license/dmahajan980/covid19"></a>
 
 A command-line interface to check the current statistics of COVID-19 of the world or a particular country.
 
-## TABLE OF CONTENTS
-1. [Getting Started](#getting-started)
-2. [Usage](#usage)
-3. [Contributing](#contributing)
-5. [License](#license)
+## Table of Contents
 
-## GETTING STARTED
+1.  [Getting Started](#getting-started)
+2.  [Usage](#usage)
+3.  [Contributing](#contributing)
+4.  [License](#license)
 
-1. Clone or download the repository.
+## Getting Started
 
-2. Install all the dependencies using the following command
+Getting started with the CoVID-19 CLI is very simple. Kindly follow the steps given below,
 
-   ```bash
-   npm install
-   ```
-   
-3. Create a symlink of the repository in the global folder
+1.  Clone or download the repository using the command
 
-   ```bash
-   sudo npm link
-   ```
+    ```bash
+    git clone https://github.com/dmahajan980/covid19.git
+    ```
 
-## USAGE
+2.  Install all the dependencies using the following command
 
-The CLI will return the latest global statistics about COVID-19 in a tabular format.
+    ```bash
+    npm install
+    ```
 
-   ```bash
-   covid19 stats
-   ```
+3.  Create a symlink of the repository in the global folder
 
-The CLI can also return the COVID-19 statistics of a country, if specified, using one of the following commands.
+- Windows
+  If you are on Windows, start `cmd.exe` as Administrator and run the command
 
-   ```bash
-   covid19 stats --country <countryName>
-   ```
+  ```bash
+  npm link
+  ```
 
-   ```bash
-   covid19 stats -c <countryName>
-   ```
+- macO/Linux
+  If you are on macOS/Linux, the command is
+  ```bash
+  sudo npm link
+  ```
 
-The supported formats for `countryName` are ISO2, ISO3 or a single-word name.
+## Usage
 
-## CONTRIBUTING
+This CLI is very easy to use.
+
+- **`covid19`**  
```

Comment:
Sorry for not adding this earlier, but I feel that you can improve this by making this a subheading instead of a bullet point. So, in that case, this line would be

```suggestion
### `covid19`  
```

Reply:
Also, don't forget to remove the indentation (due to the bullet point) of the contents under the subsection.